person.rb

```diff
@@ -11,8 +11,13 @@ def initialize(age, name = 'Unknown', parent_permission = "true")
     @name = name
     @age = age 
     @parent_permission = parent_permission
+    @rentals = []
   end 
 
+  def rent_book(rental)
+    @rentals.push(rental)
+end
```

Comment:
- Hey, let's also improve on the `rent_book/ add_rental` method as we should be able to keep track of the date and the book rented by a person. As this helps in establishing the relationship between the Person and a Book initiated by the Rentals, which is why the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
```
 def add_rental(book, date)
    Rental.new(date, self, book)
  end
```

Reply:
@zieeco Thank you so much for this tips!